src/Html.php

```diff
@@ -1625,6 +1625,43 @@ public static function cssStyleToArray(string $style): array
 
     /**
      * Returns the real attribute name from the given attribute expression.
+     * If `$attribute` has neither prefix nor suffix, it will be returned back without change.
+     * @param string $attribute the attribute name or expression
+     * @return string the attribute name without prefix and suffix.
+     * @throws InvalidArgumentException if the attribute name contains non-word characters.
+     * @see Html::parseAttribute()
+     */
+    public static function getAttributeName(string $attribute): string
+    {
+        return static::parseAttribute($attribute)['name'];
+    }
+
+    /**
+     * @param string $wrapName
+     * @param string $attributeName
+     * @return string
+     * @throws InvalidArgumentException if the attribute name contains non-word characters.
+     * @throws EmptyWrapNameException if wrap name is empty and attribute name is tabular.
+     */
+    public static function wrapAttributeName(string $wrapName, string $attributeName): string
```

Comment:
It's not clear what the method does. What is "wrap"? Also, based on the code, it seems it's not wrapping name but prefixing it.

Reply:
I try move [this code](https://github.com/yiisoft/form/blob/67f40a2ddb6bdbb75e86ea431dff9596a239dab9/src/Helper/HtmlForm.php#L87-L101) to Html Helper... Has idea how do name the method?

Reply:
Hmm... it looks very form-packge-specific. Is there a use case for it outside of form package?

Reply:
> Hmm... it looks very form-packge-specific. Is there a use case for it outside of form package?

We have method `getAttributeName` which also work with attribute expressions such as `[0]content` (used in tabular data input to represent the "content" attribute for the first model in tabular input), `dates[0]` (represents the first array element of the "dates" attribute), `[0]dates[0]` (represents the first array element of the "dates" attribute for the first model in tabular input).

Why not have a method (`wrapAttributeName`) for this functionality?

Reply:
Hm.. It looks like `getAttributeName` also is only needed in `yiisoft/form`.